homeassistant/components/shelly/binary_sensor.py

```diff
@@ -48,13 +53,30 @@
     ),
 }
 
+REST_SENSORS = {
+    ("cloud"): RestAttributeDescription(
+        name="Cloud",
+        device_class=DEVICE_CLASS_CONNECTIVITY,
+        path="cloud/connected",
+    ),
+    ("fwupdate"): RestAttributeDescription(
+        name="Firmware update (release)",
```

Comment:
IMO "Firmware update (release)" name is too long and confusing. Maybe "Update available"?

Reply:
I'm going to add also "Firmware update (beta)" as soon as I have a device that offer a new firmware so to check code is working.
What you suggest ?

Reply:
Honestly, I'm not convinced that we need a sensor informing about the beta version of the firmware. That's what the device's web panel is for. Second opinion needed.

Reply:
I agree. In fact I don't think that we should have a firmware update available at all. That is something we should be able to do via device registry.

Reply:
Ok, so which is the final name we want here ?

Reply:
We should remove it.

Reply:
I applied the name suggestion, even if we do't have an agreement if we leave or not the sensor.